### ISSUE TITLE
Anonymized private activity

### ISSUE THREAD

**Issue by make-github-pseudonymous-again** [go-gitea/gitea#9546]
This is a feature request.

Feature: to be able to publicize activity on private repositories in a way that does not reveal information on what is being worked on. For example by adding this activity to the activity graph. Currently only public activity is shown on the activity graph.

Similar to this github feature: https://help.github.com/en/github/setting-up-and-managing-your-github-profile/publicizing-or-hiding-your-private-contributions-on-your-profile#changing-the-visibility-of-your-private-contributions

**Comment by aureooms** [comment]
Maybe duplicate of #9447?

Related to #6, #2349, #7392.

**Comment by bagasme** [comment]
@aureooms redacting activity?

**Comment by aureooms** [comment]
Maybe there is a misunderstanding (see #9447). Is activity in private repositories already supposed to be part of the activity graph already? Because I cannot see these activities on my instance.

**Comment by hell001** [comment]
Yup. Per OPs [link](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile#changing-the-visibility-of-your-private-contributions), it would be appreciated to be able to show that contributions are being made, even if the repos are not public. (especially useful to those with many private repos...)
